src/clj/runbld/notifications.clj

```diff
@@ -0,0 +1,37 @@
+(ns runbld.notifications
+  (:require [runbld.schema :refer :all]
+            [pallet.thread-expr :refer [when->]]
+            [clojure.string :as str]
+            [runbld.util.date :as date]
+            [schema.core :as s]))
+
+(defn strip-out-runbld [src]
+  (let [runbld-shebang (fn [line]
+                         (not
+                          (re-find #"^#!.*runbld" line)))]
+    (->> src
+         java.io.StringReader.
+         clojure.java.io/reader
+         line-seq
+         (filter runbld-shebang)
```

Comment:
This could be `(remove #(re-find #"^#!.*runbld" %))` and then you don't need the let binding or `fn`